StreamChat_v3.xcodeproj/xcshareddata/xcschemes/DemoApp.xcscheme

```diff
@@ -50,6 +50,12 @@
             ReferencedContainer = "container:StreamChat_v3.xcodeproj">
          </BuildableReference>
       </BuildableProductRunnable>
+      <CommandLineArguments>
+         <CommandLineArgument
+            argument = "-UIViewLayoutFeedbackLoopDebuggingThreshold 100"
```

Comment:
I think we should keep this flag for future loop cases

Reply:
👍